tests/safeds/data/tabular/containers/_table/test_remove_rows_with_outliers.py

```diff
@@ -45,11 +69,17 @@
                     "col2": [],
                 },
             ),
-            0,
+            Table(
+                {
+                    "col1": [],
+                    "col2": [],
+                },
+            ),
         ),
     ],
     ids=["no outliers", "with outliers", "no rows"],
 )
-def test_should_remove_rows_with_no_outliers(table: Table, expected: int) -> None:
+def test_should_remove_rows_with_no_outliers(table: Table, expected: Table) -> None:
     updated_table = table.remove_rows_with_outliers()
-    assert updated_table.number_of_rows == expected
+    assert updated_table.schema == expected.schema
+    assert updated_table.number_of_rows == expected.number_of_rows
```

Comment:
```suggestion
    assert updated_table.number_of_rows == expected.number_of_rows
    assert updated_table == expected
```

Reply:
This line is important, because the test verifies, if rows with outliers  are removed. This can not be tested by comparing the schemas only.

Reply:
This new assertion doesn't seem to work properly on empty tables. It throws an assertion error although both tables are equal. I'll address this next friday.
<img width="1251" alt="Screenshot 2023-05-24 at 12 40 08" src="https://github.com/Safe-DS/Stdlib/assets/91055000/0bca0888-4037-4157-81ac-0d36312a155e">

Reply:
@daniaHu I don´t want to remove an assertion. It just shows it as "remove this line and add the exact same at the exact same position" because you cannot add lines in a suggestion in between lines (at least as far as i know)

@alex-senger This will be fixed when #314 is merged